Kudu.Services/FetchHelpers/OneDriveHelper.cs

```diff
@@ -62,7 +62,7 @@ internal ILogger Logger
             ChangeSet changeSet = null;
             string cursor = _settings.GetValue(CursorKey);
             ChangesResult changes = null;
-            using (_tracer.Step("Getting delta changes with cursor: {0} ...", cursor))
+            using (_tracer.Step("Getting delta changes with cursor: {0}...", cursor.Truncate(5)))
```

Comment:
ah.. i see. this is the very first trace, that is why file name same as title content.
i would suggest to change to below, so that the cursor will go into content instead of showing up in file name.

````
            using (_tracer.Step("Getting delta changes"))
            using (_tracer.Step("cursor: {0} ...", cursor))
            {
                changes = await GetChanges(info.TargetChangeset.Id, info.AccessToken, info.RepositoryUrl, cursor);
            }
````